skopt/tests/test_gp_opt.py

```diff
@@ -71,3 +73,17 @@ def test_gpr_default():
     """Smoke test that gp_minimize does not fail for default values."""
     gp_minimize(branin, ((-5.0, 10.0), (0.0, 15.0)), n_random_starts=1,
                 n_calls=2)
+
+
+@pytest.mark.fast_test
+def test_use_given_estimator():
+    """ Check that gp_minimize use the given estimator. """
+    domain = [(1.0, 2.0), (3.0, 4.0)]
+    space = normalize_dimensions(domain)
```

Comment:
do we need this line?

Reply:
I have to give a Space to cook_estimator so I think it's needed

Reply:
`cook_estimator("GP", [(1.0, 2.0), (3.0, 4.0)], noise=1234)` also works no? To save the call to `normalize_dimensions`